mobile/src/bundles/talent/talent-api.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiPath } from '~/bundles/common/enums/enums';
+// import { ApiPath, ContentType } from '~/bundles/common/enums/enums';
+// import { UserDetailsApiPath } from '~/bundles/talent/enums/enums';
+import { type ProfileStepDto } from '~/bundles/talent/types/types';
+import { HttpApiBase } from '~/framework/api/api';
+import { type Http } from '~/framework/http/http';
+import { type Storage } from '~/framework/storage/storage';
+
+type Constructor = {
+    baseUrl: string;
+    http: Http;
+    storage: Storage;
+};
+
+//TODO update when UserDetailsApiPath is ready
+class TalentApi extends HttpApiBase {
+    public constructor({ baseUrl, http, storage }: Constructor) {
```

Comment:
```suggestion
    public constructor({ baseUrl, http, storage }: Constructor) {
```
Are you sure that we need storage here?

Reply:
I'm not sure.
I thought it would be better to have everything related to onboarding in the talents folder... 

And I see that the backend is preparing **ApiPath.USER_DETAILS** and **UserDetailsApiPath.UPDATE** to update this data, so I guess we need a separate talent-api.ts 

But while UserDetailsApiPath is not yet available, I left this code
```
public constructor({ baseUrl, http, storage }: Constructor) {
        super({ path: ApiPath.AUTH, baseUrl, http, storage })
    }
```
so that no error occurs.